### PR TITLE
JavaScript sample snippet to demonstrate sending an enter to an arbitrary element

### PR DESCRIPTION
**Pull request by bertold**
This was inspired by this thread on Friends of mabl:
https://mablusers.slack.com/archives/C69542NBS/p1567706267003800

### CODE REVIEW

File: mabl snippets/sendEnter.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * the enterTargetSelector variable.
+ * 
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+  let enterEvent = new KeyboardEvent("keydown", {
+    bubbles: true, cancelable: true, keyCode: 13
```
Comment (amendelsohn) [inline mabl snippets/sendEnter.js:17]: is it worth adding a comment here with a link to a doc with keycodes so people can more easily update the script to suit their needs?

File: mabl snippets/sendEnter.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Sends an enter key press to an element specified using a selector in
+ * the enterTargetSelector variable.
```
Comment (amendelsohn) [inline mabl snippets/sendEnter.js]: Might be nice to separate out the prerequisite variables. Maybe make a section for the variables and what to put in them.

```
/*
 * required mabl variables:
 *   enterTargetSelector - an XPath/css selector to locate the target element
 * ...
 */
```

**Review by amendelsohn — APPROVED** [review]
This is awesome! Thank you 😄

**Review by DonMcNamara — APPROVED** [review]
Nice!

File: mabl snippets/sendEnter.js (hunk 1 above)
Comment (bertold) [reply mabl snippets/sendEnter.js:17]: Great idea. I was thinking about adding a link to https://keycode.info/

File: mabl snippets/sendEnter.js (hunk 2 above)
Comment (bertold) [reply mabl snippets/sendEnter.js]: Great idea.

**Pull request merged**